tests/unit/test.program.js

```diff
@@ -32,7 +36,9 @@ describe('program.Program', () => {
     let thing = spy(() => Promise.resolve());
     let program = new Program(['thing'])
       .command('thing', 'does a thing', thing);
-    return execProgram(program)
+    return execProgram(program, {
+      checkForUpdates: spy(),
```

Comment:
Passing this directly creates unnecessary repetition. I don't see what the problem is with adding it as a default to [execProgram()](https://github.com/shubheksha/web-ext/blob/305f83ffff28cb1bb38aeb91d7636b1bd97559b0/tests/unit/test.program.js#L30). What problem were you running into?